bot/slack/commands/ip_info.py

```diff
@@ -0,0 +1,49 @@
+import netaddr
+import os
+
+from mozdef_util.geo_ip import GeoIP
+
+
+def is_ip(ip):
+    try:
+        netaddr.IPNetwork(ip)
+        return True
+    except Exception:
+        return False
+
+
+def ipLocation(ip):
```

Comment:
For consistency, let's rename this to `is_location` as per [PEP 8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names).

Reply:
Fixed in https://github.com/mozilla/MozDef/pull/1018/commits/e05243983cb9167303a19e85a3c88f74da8e2612